merge_requests.go

```diff
@@ -328,6 +328,27 @@ func (s *MergeRequestsService) GetMergeRequestChanges(pid interface{}, mergeRequ
 	return m, resp, err
 }
 
+// CreateMergeRequest Get all the issues that would be closed by merging the provided merge request.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ce/api/merge_requests.html#list-issues-that-will-close-on-merge
+func (s *MergeRequestsService) ClosesIssues(pid interface{}, mr_iid int, options ...OptionFunc) ([]*Issue, *Response, error) {
```

Comment:
I don't think `ClosesIssues` is a good (not consistent) name. Maybe make it `GetIssuesClosedOnMerge`? It's a bit long, but very descriptive and more inline with the other function names.

Reply:
And please rename `mr_iid` to mergeRequest` as it's called in other functions.